artemis-protocols/artemis-stomp-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/stomp/v11/StompFrameV11.java

```diff
@@ -66,7 +66,7 @@ public ActiveMQBuffer toActiveMQBuffer() throws Exception
             head.append(h.getEncodedValue());
             head.append(Stomp.NEWLINE);
          }
-         if (bytesBody != null && bytesBody.length > 0)
+         if (bytesBody != null && bytesBody.length > 0 && !hasHeader(Stomp.Headers.CONTENT_LENGTH))
```

Comment:
Does this introduce any form of a security issue? What happens on the receiver side if the content length doesn't match the header?